doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -158,19 +158,20 @@ Functional requirements of this node include:
 
 <a name="4.3.2"></a>
 ### 4.3.2 Overview
-The following OpenStack components are deployed on the Infrastructure. Some of them will be only deployed within control plane and some of them will be deployed within both control and compute nodes:
-
-| Service  |  Description  | Deployed on Controller Nodes | Deployed on Compute Nodes |
-|-----------|-------------|-----------|-----------------|
-| Keystone  |  the authentication service  | X |  |
-| Glance  |  the image management service  | X |  |
-| Cinder  |  the block storage management service  | X |  |
-| Swift  |  the Object storage management service  | X | X |
-| Neutron  |  the network management service  | X | X |
-| Nova  |  the compute resources management service  | X | X |
-| Ironic  | the Bare Metal Provisioning service  | X | X |
-| Heat  |  the orchestration service  | X |  |
-| Horizon  |  the WEB UI service  | X |  |
+The following OpenStack components are deployed on the Infrastructure. Some of them will be only deployed on control hosts and some of them will be deployed within both control and compute hosts. The Table also maps the OpenStack core services to the Reference Model Management Software components (<a ref="https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter03.md#3.3">Reference Model Chapter 3.3 Management Software</a>).
+
+| RM Management Software| Service| Description| Required / Optional| Deployed on Controller Nodes| Deployed on Compute Nodes |
```

Comment:
Can you please explain what is the meaning of the "RM Management Software" column? For me it is confusing, sorry :)
We should make a decision if we include Palcement or not (and it is also a decision on the version range of OpenStack).

Reply:
@CsatariGergely Good catch -- will add the acronym "RM" in the text above the Table -- "Reference Model (RM) Management Software" 

W.r.t. Placement, it is introduced in the Stein OpenStack release -- we have had previous discussions but no decisions on how current should our version of documents be. Currently, they are rooted on the Pike release.

Reply:
> 
> 
> @CsatariGergely Good catch -- will add the acronym "RM" in the text above the Table -- "Reference Model (RM) Management Software"
> 
> W.r.t. Placement, it is introduced in the Stein OpenStack release -- we have had previous discussions but no decisions on how current should our version of documents be. Currently, they are rooted on the Pike release.

Thanks, but even with explaining what RM is I have problems understanding what is the purpose of the column. 
If we decide to use Pike we should mention that in the specs.